modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2147,10 +2147,7 @@ export class Wallet implements IWallet {
     const signingParams = {
       ...params,
       txPrebuild,
-      wallet: {
-        // this is the version of the multisig address at wallet creation time
-        addressVersion: this._wallet.coinSpecific.addressVersion,
```

Comment:
this was used here, but the current `algo` code doesn't use it anymore https://github.com/BitGo/BitGoJS/commit/7a02a19745035ec6851a1eaf478249c59be86068#diff-8a72caea304a885f750f24d950ccf6a0e5a8372d9f5c941d1856154d029ce6caR140